draft-ietf-snac-simple.xml

```diff
@@ -488,16 +488,31 @@
 	    isn't worth doing.]</t>
 	</section>
 	<section>
-	  <name>Generating a ULA prefix to provide addressability</name>
+	  <name>Generating a per-stub-router ULA site prefix</name>
 	  <t>
-	    In order to be able to provide addressability either on the stub network or on an adjacent infrastructure network, a stub
-	    router must allocate its own ULA prefix. ULA prefixes, described in Unique Local IPv6 Unicast Addresses
-	    (<xref target="RFC4193"/>) are randomly allocated prefixes. A stub router MUST allocate a single ULA prefix for use in
-	    providing on-link prefixes to the stub network and the infrastructure network, as needed.
+	    Some technologies used for stub networks, for example Thread or 6LoWPAN mesh networks, can produce partitioned networks,
+            where what is notionally the same stub network winds up looking like two or more discrete links. For mesh networks, such
+            partitions can form and rejoin over time as a result of either changes in radio propagation or the addition of, or
+            removal of, devices on the mesh.
 	  </t><t>
-	    The ULA prefix allocated by a stub router SHOULD be maintained across reboots, and SHOULD remain stable over
-	    time. For privacy reasons, a stub router that roams from network to network may wish to allocate a different ULA prefix
-	    each time it connects to a different infrastructure network.
+	    On stub networks that can partition, one way of detecting that a partition has occurred is to notice that the stub
+	    router that has advertised the on-link prefix for the stub network is no longer reachable. When this occurs, stub
+	    routers that notice this loss of reachability MUST advertise a ULA link prefix derived from their ULA site prefix on the
+	    stub network.
+	  </t><t>
+	    An implication of this is that when such a partition forms, the same ULA link (?) prefix can’t be advertised on both
+	    partitions, since this will result in ambiguous routing. We address this problem by requiring that each stub router
+	    generate its own ULA site prefix. This prefix is then available for two purposes: providing addressing on the AIL, if
+	    needed, and providing addressing on the stub network, if needed.
+	  </t><t>
+	    A stub router must therefore allocate its own ULA site prefix. ULA prefixes, described in Unique Local IPv6 Unicast
+            Addresses (<xref target="RFC4193"/>), are randomly allocated prefixes.  A stub router MUST allocate a single ULA site
```

Comment:
Length of ULA site prefix is not defined. RFC 4193 doesn't define "ULA site prefix".

If every stub router creates its own ULA site prefix, how can we get to the case that multiple stub routers will use the same ULA /64 prefix for the AIL as was intended by PR #39 ?

Reply:
The problem is that RFC 4193 doesn't really make a distinction between a ULA /48 and a ULA /64 that's derived from that ULA /48. That is, there is no terminology for the two types of ULA prefix. What we mean by "site prefix" is "ULA /48," or "the prefix identified by the ULA global identifier."

I think the way to address this is to just define the term in the glossary. I'd say something like:

ULA site prefix: a ULA /48 prefix from which allocations can be made.

ULA subnet prefix: a ULA /64 derived from a particular ULA site prefix.

Reply:
Added "ULA Site Prefix" and "ULA Link Prefix" to Glossary section - terms that were proposed during the last SNAC meeting.